lib/grape/validations/attributes_doc.rb

```diff
@@ -29,6 +29,7 @@ def extract_details(validations)
 
         details[:default] = validations[:default] if validations.key?(:default)
 
+        details[:exact_length] = validations[:length][:exact] if validations.key?(:length) && validations[:length].key?(:exact)
```

Comment:
Will undo this change bc this details key is not part of the swagger specs. 
REF: https://swagger.io/docs/specification/data-models/data-types#string